Description: installed arpwatch's *.awk scripts into /usr/lib/arpwatch
 and edited massagevendor to set AWKPATH before running awk
Author: Craig Sanders <cas@taz.net.au>

diff --git a/massagevendor b/massagevendor
index 9b1eaaa..e135820 100755
--- a/massagevendor
+++ b/massagevendor
@@ -9,6 +9,7 @@
 #
 # - Deal with duplicates in oui.txt (concatenate company names)
 #
+export AWKPATH="$AWKPATH:/usr/lib/arpwatch"
 (sed -n \
     -e 's/^\([0-9A-F][0-9A-F]\)-\([0-9A-F][0-9A-F]\)-\([0-9A-F][0-9A-F]\)  *(hex)[ 	]*\(..*\)/\1\2\3	\4/p' \
     $* | \
